truss/tests/test_custom_server.py

```diff
@@ -19,7 +19,7 @@ def test_custom_server_truss(test_data_path):
         try:
             print("Starting container")
             _, urls = tr.docker_run_for_test(
-                model_server_stop_retry_override=stop_after_attempt(3)
+                model_server_stop_retry_override=stop_after_attempt(10)
```

Comment:
Locally this truss seemed to need a little more time to load, consistently worked at `5` for me but I don't see a downside to having 10. In general I'm not sure why this needs an override vs the default 120s, but didn't want to investigate that now